cumulusci/oauth/salesforce.py

```diff
@@ -131,7 +157,16 @@ def do_GET(self):
             http_body = f"error: {args['error'][0]}\nerror description: {args['error_description'][0]}"
         else:
             http_status = http.client.OK
-            http_body = "Congratulations! Your authentication succeeded."
+            gif = random.choice(
+                [
+                    "tMUMi4Ylnzk88",
+                    "Cj3Ce7e8h2EKY",
+                    "3MibWRcT0AOYG7HUeT",
+                    "9uoYC7cjcU6w8",
+                ]
+            )
+            http_body = f"""<html><p>Congratulations! Your authentication succeeded.</p>
+            <iframe src="https://giphy.com/embed/{gif}" width="480" height="360" frameBorder="0"></iframe></html>"""
```

Comment:
This is maybe a bit too cute. The 2nd gif is already a 404.

Reply:
The last one is just a cut and paste error in the PR (fixed above) and I don't think Giphy deletions are common.

https://medium.com/@makeagif/http-help-makeagif-com-common-makeagif-q-s-will-my-gifs-be-deleted-2a0de1e75a57

The 99% of users who are "free users" have to ask to get files deleted through a "support request".

https://giphy.com/embed/Cj3Ce7e8h2EKY

Reply:
Still not a fan of embedding something from a domain we don't control.

Reply:
4a0b926